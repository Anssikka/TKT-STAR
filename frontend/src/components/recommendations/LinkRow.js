import React from 'react'
import { Link } from 'react-router-dom'
import Row from '../common/Row'
import '../../styles/Button.css'

function LinkRow() {
  return (
    <Row>
      <Link className="button-reverse" to="/recommendations/books">
        Add a Book
      </Link>
      <Link className="button-reverse" to="/recommendations/videos">
        Add a Video
      </Link>
      <Link className="button-reverse" to='/recommendations/blogs'>
        Add A Blog
        </Link>
    </Row>
  )
}

export default LinkRow
