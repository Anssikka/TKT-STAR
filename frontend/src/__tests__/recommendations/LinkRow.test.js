import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render, cleanup, fireEvent, waitForElement } from '@testing-library/react'
import LinkRow from '../../components/recommendations/LinkRow'
import { BrowserRouter as Router, Route } from 'react-router-dom'

afterEach(cleanup)

describe('<Book />', () => {
  it('has correct links', () => {
    const component = render(
      <Router>
        <Route render={() => <LinkRow />} />
      </Router>
    )
    expect(component.asFragment()).toMatchSnapshot()
  })
})
