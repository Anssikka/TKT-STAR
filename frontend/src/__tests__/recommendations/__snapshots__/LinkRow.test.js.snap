// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Book /> has correct links 1`] = `
<DocumentFragment>
  <div
    class="row container"
  >
    <a
      class="button-reverse"
      href="/recommendations/books"
    >
      Add a Book
    </a>
    <a
      class="button-reverse"
      href="/recommendations/videos"
    >
      Add a Video
    </a>
  </div>
</DocumentFragment>
`;
